azurerm/internal/services/eventgrid/tests/eventgrid_domain_topic_resource_test.go

```diff
@@ -0,0 +1,156 @@
+package tests
+
+import (
+	"fmt"
+	"net/http"
+	"testing"
+
+	"github.com/hashicorp/terraform-plugin-sdk/helper/resource"
+	"github.com/hashicorp/terraform-plugin-sdk/terraform"
+	"github.com/terraform-providers/terraform-provider-azurerm/azurerm/internal/acceptance"
+	"github.com/terraform-providers/terraform-provider-azurerm/azurerm/internal/clients"
+	"github.com/terraform-providers/terraform-provider-azurerm/azurerm/internal/features"
+	"github.com/terraform-providers/terraform-provider-azurerm/azurerm/utils"
+)
+
+func TestAccAzureRMEventGridDomainTopic_basic(t *testing.T) {
+	data := acceptance.BuildTestData(t, "azurerm_eventgrid_domain_topic", "test")
+
+	resource.ParallelTest(t, resource.TestCase{
+		PreCheck:     func() { acceptance.PreCheck(t) },
+		Providers:    acceptance.SupportedProviders,
+		CheckDestroy: testCheckAzureRMEventGridDomainTopicDestroy,
+		Steps: []resource.TestStep{
+			{
+				Config: testAccAzureRMEventGridDomainTopic_basic(data),
+				Check: resource.ComposeTestCheckFunc(
+					testCheckAzureRMEventGridDomainTopicExists(data.ResourceName),
+					resource.TestCheckResourceAttrSet(data.ResourceName, "id"),
+				),
+			},
+			data.ImportStep(),
+		},
+	})
+}
+
+func TestAccAzureRMEventGridDomainTopic_requiresImport(t *testing.T) {
+	if !features.ShouldResourcesBeImported() {
+		t.Skip("Skipping since resources aren't required to be imported")
+		return
+	}
```

Comment:
_minor_ could we remove this? this feature flag is no longer necessary since this is on by default in 2.0 (we're going through cleaning this up gradually)